messaging/broker_test.go

```diff
@@ -124,13 +129,19 @@ func TestBroker_Apply_SetMaxTopicIndex(t *testing.T) {
 	if err := b.Apply(&messaging.Message{
 		Index: 2,
 		Type:  messaging.SetTopicMaxIndexMessageType,
-		Data:  []byte{0, 0, 0, 0, 0, 0, 0, 20, 0, 0, 0, 0, 0, 0, 0, 5}, // topicID=20, index=5
+		Data: []byte{0, 0, 0, 0, 0, 0, 0, 20, 0, 0, 0, 0, 0, 0, 0, 5, // topicID=20, index=5,
+			0, 26, 104, 116, 116, 112, 58, 47, 47, 108, 111, 99, 97, 108, //len=26, url="http://localhost:1234/data"
```

Comment:
I'll take your word for it. :-)

We could have made this slice, or at least part of it, but appending `[]byte("http://")`.